app/src/main/java/ar/com/wolox/android/cookbook/CookbookApplication.kt

```diff
@@ -31,6 +36,8 @@ class CookbookApplication : WolmoApplication() {
             androidContext(this@CookbookApplication)
             initializeModules()
         }
+
+        notificationChannelFactory.init(this)
```

Comment:
Why don't you avoid this by getting the context injected on `NotificationChannelFactory` constructor?